sci-libs/opencascade/opencascade-0.18.3.recipe

```diff
@@ -0,0 +1,198 @@
+SUMMARY="A C++ 3D modeling library"
+DESCRIPTION="Open CASCADE Technology (OCCT) is an object-oriented C++ class \
+library designed for rapid production of sophisticated domain-specific \
+CAD/CAM/CAE applications."
+HOMEPAGE="https://github.com/tpaviot/oce/"
+COPYRIGHT="1993-1999 Matra Datavision
+	1999-2014 OPEN CASCADE SAS"
+LICENSE="GNU LGPL v2.1
+	OCCT_LGPL_EXCEPTION"
```

Comment:
It have a nice name, so I would suggest using it, or at least use spaces instead of those underscores.

```diff
-	OCCT_LGPL_EXCEPTION"
+	Open CASCADE exception v1.0"